docs/4.0. Behaviour.md

```diff
@@ -181,7 +181,7 @@ Each channel contains two fields - `input` and `channel_index`. `channel_index`
 
 If no Input channel is being routed to an Output channel, both `input` and `channel_index` MUST be set to `null`. Any attempt to patch one of the two fields within a Output channel object to `null` without setting the other SHOULD result in an HTTP 400 response. In this error scenario the API SHALL NOT modify the map resource in any way, even if some of the routes in the failed request would have been valid. If an activation request is included in a failed stage request that activation MUST be ignored.
 
-Where an Output does not have a channel routed to it:
+Where an Output does not have a channel routed to it, it MUST interpret the absence of the channel in one of the following ways:  [or are other interpretations permissible?]
```

Comment:
Not as far as I've been able to think of - but I wonder if this might be better off being a "SHOULD"? Not behaving like this won't break interoperability, and it's possible a manufacturer would have a good reason for their device not behaving like this?